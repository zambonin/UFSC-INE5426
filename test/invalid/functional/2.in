# wrong number of parameters on lambda function
int a = 2, b = 4, d, c = 3
lambda int x, y -> x + y
d = Î»(a, b, c)
