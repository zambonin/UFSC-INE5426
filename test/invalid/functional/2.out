[Line 4] semantic error: function lambda expects 2 parameters but received 3
